draft-enghardt-panrg-path-properties.md

```diff
@@ -96,19 +96,19 @@ Link:
 : A medium or communication facility that connects two or more nodes with each other and enables them to exchange packets on a specific layer. A link can be physical, e.g., a WiFi network which connects an Access Point to one or multiple stations, or virtual, e.g., a virtual switch which connects two virtual machines hosted on the same physical machine.
 
 Path element:
-: Either a node or a link on a specific layer. For example, a router implementing IPv6 and Ethernet may be a path element on a network layer path, a link layer path, and a physical layer path. If this router does not implement transport layer functionality, it is hidden to any higher layer path, e.g., a transport layer path between two hosts. The link connecting these two hosts may be considered a virtual link and may abstract different physical links for each packet sent over the path.
+: Either a node or a link. For example, a router implementing IPv6 may be a path element on a path when considering the network layer. If this router does not implement transport layer functionality, it is hidden when a higher layer is considered, e.g., the transport layer.
```

Comment:
I would rephrase the second sentence:

If this router does not implement transport layer functionality, it is hidden when a higher layer, such as the transport or application layer, is considered.

Reply:
Agreed, if we keep the layering at all -- I'm not too happy with the whole "when considering the XYZ layer", but I think the concept of "considering a path on XYZ layer" is better than defining a separate path for each layer.